packages/insomnia-app/app/ui/components/wrapper-design.js

```diff
@@ -155,12 +154,84 @@ class WrapperDesign extends React.PureComponent<Props, State> {
     }
   }
 
+  renderEditor = (): React.Node => {
```

Comment:
Would prefer regular function syntax (prefixed with `_` for private functions) here to keep consistent with the rest of the class methods 😄 same with `renderPreview` below and some other instances in this PR. There's probably an ES Lint rule to govern this that we should enable...

```suggestion
  _renderEditor(): React.Node {
```